quinn-proto/src/connection/paths.rs

```diff
@@ -154,6 +156,7 @@ impl PathData {
             observed_addr_sent: false,
             last_observed_addr_report: None,
             status: Default::default(),
+            status_seq_no: VarInt::default(),
```

Comment:
Is this correct?  I would think a peer is allowed to use 0 as initial sequence number and then you'd be ignoring the first such frame.

Reply:
Is it the worst?
https://play.rust-lang.org/?version=stable&mode=debug&edition=2024&gist=2d4e756d1ffe07715c8de29db6c86e46